fairscale/nn/model_parallel/cross_entropy.py

```diff
@@ -27,6 +27,7 @@
 
 
 class _VocabParallelCrossEntropy(torch.autograd.Function):
+    # FIXME: docs
```

Comment:
good idea. the whole model_parallel dir is an import from nvidia megatron code if I am not mistaken. We again need to determine how to keep it. Maybe we want to keep it to make sure a version of it work seamlessly with fsdp or pipeline, or for other reasons. I don't really know.